src/astro/databases/google/bigquery.py

```diff
@@ -236,3 +257,177 @@ def gs_to_bigquery(
         self.hook.insert_job(
             configuration=job_config,
         )
+
+    def s3_to_bigquery(
+        self,
+        source_file: File,
+        target_table: Table,
+        if_exists: LoadExistStrategy = "replace",
+        native_support_kwargs: Optional[Dict] = None,
+        **kwargs,
+    ):
+        """
+        Transfer data from S3 to Bigquery via invoking datatransfer job
```

Comment:
```suggestion
        Load content of multiple files in S3 to output_table in Bigquery by using a datatransfer job.
```

We need to add a link to the data transfer job service! Also, we should say that there are costs associated to this transaction.